aemedge/scripts/aem.js

```diff
@@ -650,18 +650,6 @@ function decorateBlocks(main) {
   main.querySelectorAll('div.section > div > div').forEach(decorateBlock);
 }
 
-/**
```

Comment:
I would not remove this from aem.js. just leave it as is. we just don't call it for now.